### DISCUSSION TITLE
how to parse record in to `visible` function ?

### DISCUSSION THREAD

**Discussion by maherelgamil** [filamentphp/filament#4583] [Help] [answered]
I am trying to parse $record in visible or hidden functions of columns, but not working 

```php

->visible(fn($record) => $record->type !== Project::TYPE_SURA)

```

Any way to do that?!

**Comment by maherelgamil** [comment]
I am trying it o different projects and the same result, i will write an issue with this ASAP.

**Comment by danharrin** [comment, marked as answer]
The record isnt available in there, because you cant hide a column for some records and not others, thats now how tables work
